homeassistant/components/connector/cover.py

```diff
@@ -0,0 +1,295 @@
+"""Support for Motion Blinds using their WLAN API."""
+
+import logging
+
+from connectorlocal.connectorlocal import (
+    ONEWAYWIRELESSMODE,
+    TWOWAYWIRELESSMODE,
+    VENETIANTYPE,
+    WIFIMOTORTYPE,
+)
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    ATTR_TILT_POSITION,
+    CoverDeviceClass,
+    CoverEntity,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, KEY_COORDINATOR, KEY_GATEWAY, MANUFACTURER
+
+POSITION_DEVICE_MAP = {
```

Comment:
This is not a good name IMHO, maybe `HUB_TYPE_DEVICE_CLASS_MAP` would be better?

Reply:
Thanks for you advice，At the beginning, I just wanted to compare it with the class in the APP, so I named it